include/tensorwrapper/buffer/contiguous.hpp

```diff
@@ -47,6 +47,8 @@ class Contiguous : public Replicated {
     /// Type of a read-only reference to an object of type element_type
     using const_reference = const element_type&;
 
+    using elements_type = std::vector<element_type>;
```

Comment:
`element_vector`? IMHO `elements_type` is too close to `element_type` (in fact I missed the 's' the first time).

Reply:
`element_vector` is good with me.